content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,223 @@
+> No statistically significant differences were observed between groups in either women or men for more than 30 outcomes. However, women who had been fed soy formula reported slightly longer duration of menstrual bleeding [...], with no difference in severity of menstrual flow. They also reported greater discomfort with menstruation (unadjusted relative risk for extreme discomfort vs no or mild pain[...]).  
+- [Exposure to Soy-Based Formula in Infancy and Endocrinological and Reproductive Outcomes in Young Adulthood - 2001](https://web.archive.org/web/20191002190445/https://jamanetwork.com/journals/jama/fullarticle/194106)
+
+### Overall
+
+#### Soy Isoflavones: A Safety Review - 2003
+
+>  Whereas results in some studies are limited or conflicting, when viewed in its entirety, the current literature supports the safety of isoflavones as typically consumed in diets based on soy or containing soy products.  
+> - [Soy Isoflavones: A Safety Review - 2003](http://web.archive.org/web/20191002184755/https://www.researchgate.net/profile/Melody_Harwood2/publication/10855804_Soy_Isoflavones_A_Safety_Review/links/5bd721a8a6fdcc3a8dae5fec/Soy-Isoflavones-A-Safety-Review.pdf?origin=publication_detail)
+
+#### Soy, Soy Foods and Their Role in Vegetarian Diets - 2018
+
+This article became one of our primary sources due to its recency and broad coverage of other articles.
+
+##### Average Vegetarian Intake
+
+> Vegetarians had the major intake among the Western people, with 7-12 mg per day of total isoflavones.   
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p3](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+##### Required Active Dosage
+> Overall, the low content of bioactive compounds in second generation soy foods [(e.g. soy based mock meats)] and moderate amounts in traditional soy preparations offer modest health benefits with very limited risk for potential adverse health effects. At the same time, to have the beneficial effects of soy isoflavones, intake should be at least of 60-100 mg per day, at present not easily reached in Western countries.  
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p21](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
+##### On extrapolation from data
+> Despite the high intakes in Asian countries, there is still a paucity of long term data on soy consumption in Western countries. Currently, the use of soy foods in infancy, including soy-based infant formulas, was not linked to adverse events. However, absence of evidence is not an evidence of absence.
+> - [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018 - p22](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/)
+
```

Comment:
Perhaps need to point out again that its also due to differing types of soy products and extracts... which may all have differing effects.

Reply:
Yeah I think I'll do a section on that in this main article, which there will be room for once I've split out the pro/con articles into different sections.